OrbitGl/GlCanvas.cpp

```diff
@@ -343,6 +349,21 @@ float GlCanvas::ScreenToWorldWidth(int width) const {
   return (static_cast<float>(width) / static_cast<float>(GetWidth())) * world_width_;
 }
 
+Vec2 GlCanvas::WorldToScreen(Vec2 world_pos) const {
+  Vec2 screen_pos;
+  screen_pos[0] = (world_pos[0] - world_top_left_x_) / world_width_ * GetWidth();
+  screen_pos[1] = (world_top_left_y_ - world_pos[1]) / world_height_ * GetHeight();
+  return screen_pos;
+}
+
+int GlCanvas::WorldToScreenHeight(float height) const {
+  return static_cast<int>(height / world_height_ * GetHeight());
+}
+
+int GlCanvas::WorldToScreenWidth(float width) const {
+  return static_cast<int>(width / world_width_ * GetWidth());
+}
```

Comment:
There is an inconsistency with the WorldToScreen return type, we have both float and int for screen space coordinates. I think we should make that more uniform eventually, perhaps by using float across the board, but it's outside the scope of this PR.